app/controllers/api/v2/discovery_rules_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
           param :name, String, :required => true
           param :search, String, :required => true
           param :hostgroup_id, Integer, :required => true
-          param :hostname, String, :required => true
+          param :hostname, String
           param :max_count, Integer
```

Comment:
Isn't it confusing to have input parameter `max_count` and return the same value renamed as `hosts_limit`?